### PR TITLE
format updates to reduce use of br tags

### PR DESCRIPTION
**Pull request by cmiddi**

### CODE REVIEW

File: v1/linkml-schemas/address.yaml [hunk 1]
```diff
@@ -17,12 +17,12 @@ default_range: string
...
-      <br />
-      JSON schema - [di_vocab:PostalAddress](json-schemas/PostalAddress.json) 
-      <br />
-      <br />
+      A postal address associated with a user, returned by an Address Lookup API or by manual data input.  
+        
+
+      JSON schema - [di_vocab:PostalAddress](json-schemas/PostalAddress.json)  
+        
+
```
Comment (galund) [inline v1/linkml-schemas/address.yaml]: If the bits underneath can be in block elements then I'd imagine this could/should be too?

Comment (cmiddi) [reply v1/linkml-schemas/address.yaml]: I will have a look for some reason I could do Markdown new lines here but would be good to make it consistent.

Comment (cmiddi) [reply v1/linkml-schemas/address.yaml]: new version pushed - the first block of description cannot be surrounded by tags due to the way it is appended in hierarchical views in other pages.

Comment (galund) [reply v1/linkml-schemas/address.yaml]: Urgle, that has to be a bug, but let's roll with the bugs for now :) Can you create log an issue somewhere (in this repo I assume, but if it happened to be straightforward to report as an issue in whatever other repo then do feel free to do that).

**Review by galund — APPROVED** [review]

**Pull request merged**